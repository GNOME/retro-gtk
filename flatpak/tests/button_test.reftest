[Retro Reftest]
Path=/button_test
Core=/app/lib/libretro/button_test_libretro.so

[Frame 0]
State=None
Video=button_test.png
