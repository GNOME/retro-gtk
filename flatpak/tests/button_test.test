[Retro Reftest]
Path=/button_test
Core=/app/lib/libretro/button_test_libretro.so

[Frame 0]
Tests=Run;Video;NoStateAccess;
Video=button_test.png
