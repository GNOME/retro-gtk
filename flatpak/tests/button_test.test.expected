/button_test/Boot: OK
/button_test/0/Run: OK
/button_test/0/Video: OK
/button_test/0/NoStateAccess: OK
